src/umpire/DeviceAllocator.cpp

```diff
@@ -12,33 +12,31 @@
 
 namespace umpire {
 
-  __host__
-DeviceAllocator::DeviceAllocator(Allocator allocator, size_t size) :
-  m_allocator(allocator),
-  m_ptr(static_cast<char*>(m_allocator.allocate(size))),
-  m_size(size),
-  m_child(false)
+__host__ DeviceAllocator::DeviceAllocator(Allocator allocator, size_t size)
+    : m_allocator(allocator),
+      m_ptr(static_cast<char*>(m_allocator.allocate(size))),
+      m_size(size),
+      m_child(false)
 {
   auto& rm = umpire::ResourceManager::getInstance();
   auto device_alloc = rm.getAllocator(umpire::resource::Device);
 
-  m_counter = static_cast<unsigned int*>(
-    device_alloc.allocate(sizeof(unsigned int)));
+  m_counter =
+      static_cast<unsigned int*>(device_alloc.allocate(sizeof(unsigned int)));
   rm.memset(m_counter, 0);
 }
 
 __host__ __device__
```

Comment:
Line 29 has `__host__ __device` on it's own line while Line 39 had this changed. Should this be changed?

Reply:
I think it remains split due to line length.

Reply:
Yea, that seems likely. If you see DeviceAllocator.hpp, line 30, I think that looks a lot better. But this is ok :p